tests/platform_tests/test_reboot.py

```diff
@@ -39,9 +39,11 @@ def set_max_time_for_interfaces(duthost):
     to let MAX_TIME_TO_REBOOT to be overwritten by specified timeout value
     """
     global MAX_WAIT_TIME_FOR_INTERFACES
+    if duthost.facts["platform"] == "x86_64-cel_e1031-r0":
+        MAX_WAIT_TIME_FOR_INTERFACES = 600
     plt_reboot_ctrl = get_plt_reboot_ctrl(duthost, 'test_reboot.py', 'cold')
```

Comment:
Could you please check the comments of function [get_plt_reboot_ctrl](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/common/utilities.py#L709)? Looks like we can define the timeout for E1031 in ansible inventory file.